profiles/coreos/base/package.use.force

```diff
@@ -1,2 +1,5 @@
 # Copyright (c) 2014 The CoreOS Authors. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
+
+# Do not force this flag, we don't need XATTR_PAX
+sys-apps/portage -xattr
```

Comment:
The package.use.force removes the flag from a previous listing in package.use.force in the hardened profile as that overrides everything. The entry in package.use then can actually disable the flag since it is listed in $USE. The forced flag mechanism is basically an independent system from normal $USE and package.use system.